collectors/diskspace.plugin/plugin_diskspace.c

```diff
@@ -3,11 +3,17 @@
 #include "../proc.plugin/plugin_proc.h"
 
 #define PLUGIN_DISKSPACE_NAME "diskspace.plugin"
+#define THREAD_DISKSPACE_SLOW_NAME "PLUGIN[diskspace slow]"
 
 #define DEFAULT_EXCLUDED_PATHS "/proc/* /sys/* /var/run/user/* /run/user/* /snap/* /var/lib/docker/*"
 #define DEFAULT_EXCLUDED_FILESYSTEMS "*gvfs *gluster* *s3fs *ipfs *davfs2 *httpfs *sshfs *gdfs *moosefs fusectl autofs"
 #define CONFIG_SECTION_DISKSPACE "plugin:proc:diskspace"
 
+#define MAX_STAT_USEC 10000
```

Comment:
What is the average stat/stafs time for not network mount points? Perhaps add a comment about how you end up with 10000?

Reply:
@stelfrag, hi. @vlvkobal implemented changing update_every parameter for slow mount points based on the slowest `statvfs()` call. So the chart could be: `restart1[t0: update_every=5, ..., tn: update_every=10], restart2[t0: update_every=5, ..., tn: update_every=10], ...` Can this be a problem for the query engine/something else?

Reply:
possibly yes